src/webgpu/util/f32_interval.ts

```diff
@@ -1631,15 +1701,9 @@ export function stepInterval(edge: number, x: number): F32Interval {
   return runBinaryToIntervalOp(toF32Interval(edge), toF32Interval(x), StepIntervalOp);
 }
 
-const SubtractionInnerOp: BinaryToIntervalOp = {
-  impl: (x: number, y: number): F32Interval => {
-    return correctlyRoundedInterval(x - y);
-  },
-};
-
 const SubtractionIntervalOp: BinaryToIntervalOp = {
   impl: (x: number, y: number): F32Interval => {
-    return roundAndFlushBinaryToInterval(x, y, SubtractionInnerOp);
```

Comment:
This is a non-functional change, the inner op for subtraction is a remnant from before the framework was more built out.